src/encoder/encodersndfileflac.cpp

```diff
@@ -58,10 +58,10 @@ void EncoderSndfileFlac::setEncoderSettings(const EncoderSettings& settings)
     m_compression = static_cast<double>(settings.getCompression()) / 8.0;
 }
 
-void EncoderSndfileFlac::encodeBuffer(const CSAMPLE* pBuffer, const int iBufferSize) {
+void EncoderSndfileFlac::encodeBuffer(const CSAMPLE* pBuffer, const std::size_t bufferSize) {
     if (m_pClampBuffer) {
-        SINT numSamplesLeft = iBufferSize;
-        while (numSamplesLeft > 0) {
+            SINT numSamplesLeft = bufferSize;
+            while (numSamplesLeft > 0) {
```

Comment:
was this a confused clang-format or where is the bogus indentation from?

Reply:
I think this is not clang-format itself. I've clang-format support enabled in VisualStudio and it formats it like this:
![grafik](https://github.com/user-attachments/assets/e7d19f0d-be4d-4521-bca3-98036be29353)
but if I ran pre-commit, it looks like this:
![grafik](https://github.com/user-attachments/assets/b28f6a2d-152f-4995-bfef-80b67c3b947f)

I will not address this in this PR!

Reply:
> I will not address this in this PR!

Not forcing you to, though I wonder whether it makes sense to just enforce yourself and skip clang-format...

Reply:
This I can do, but not fixing the script

Reply:
Done! CI fails now as expected.

Reply:
yeah thats fine for me. Since it only runs on the diff we shouldn't be bothered by it again after the merge, right?